packages/components/src/components/buttonlink/buttonlink.styles.ts

```diff
@@ -0,0 +1,62 @@
+import { css } from 'lit';
+
+import { hostFocusRingStyles } from '../../utils/styles';
+
+const styles = css`
+  :host {
+    border: unset;
+  }
+
+  :host([data-btn-type='pill']),
+  :host([data-btn-type='pill-with-icon']) {
+    padding: unset;
+  }
+
+  :host([data-btn-type='pill'][size='24'])::part(anchor),
+  :host([data-btn-type='pill-with-icon'][size='24'])::part(anchor) {
+    padding: 0px 0.625rem;
+  }
+
+  :host([data-btn-type='pill'][size='32'])::part(anchor),
+  :host([data-btn-type='pill-with-icon'][size='32'])::part(anchor),
+  :host([data-btn-type='pill'][size='28'])::part(anchor),
+  :host([data-btn-type='pill-with-icon'][size='28'])::part(anchor) {
+    padding: 0px 0.75rem;
+  }
+
+  :host([data-btn-type='pill'][size='40'])::part(anchor),
+  :host([data-btn-type='pill-with-icon'][size='40'])::part(anchor) {
+    padding: 0px 1rem;
+  }
+
+  :host([disabled]) {
+    cursor: auto;
+    pointer-events: none;
+  }
+
+  /* Anchor inside ButtonLink */
+  :host::part(anchor) {
+    color: inherit;
+    font-weight: inherit;
+    text-decoration: none;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    gap: inherit;
+    height: 100%;
+    width: 100%;
+    border-radius: inherit;
+    border: 0.0625rem solid transparent;
```

Comment:
nitpick: we can use 1px here instead of rem.
```suggestion
    border: 1px solid transparent;
```